ts/Stock/StockToolsBindings.ts

```diff
@@ -419,8 +445,8 @@ bindingsUtils.attractToPoint = function (
         y: (closestPoint as any).y,
         below: y < (closestPoint as any).y,
         series: (closestPoint as any).series,
-        xAxis: (closestPoint as any).series.xAxis.index || 0,
-        yAxis: (closestPoint as any).series.yAxis.index || 0
+        xAxis: (closestPoint as any).series.xAxis.options.index || 0,
+        yAxis: (closestPoint as any).series.yAxis.options.index || 0
```

Comment:
I know this is not new to this PR, but what if `closestPoint` is undefined? That case should probably be handled. Imagine that the series has `data: []`.